contributing.md

```diff
@@ -0,0 +1,74 @@
+# How to contribute to o-icons
+
+Thank you for your interest in contributing to fticons! As our icons are used by a lot of different applications, we have a few constraints for new ones. If you aren't sure if your new icon meets any of these, please [raise an issue](http://github.com/financial-times/fticons/issues), or ask in the [#ft-origami slack channel](https://financialtimes.slack.com/messages/ft-origami/).
+Thank you!
+
+## Adding or updating an icon
+
+If you want to add or update an icon, please open a pull request, making sure the new icon meets the following criteria:
+
+### Design:
+
+1. icons must be a single colour (black)
+1. icons should be suitable for reuse in more than 1 application
+1. icons must meet a need that is not already met by a pre-existing icon
+1. icons must be square
+1. icons should align to a 40x40 pixel grid, with 10px padding on each side
+1. icons should be a solid shape rather than outlines where possible
+1. icons should have a minimum line thickness of 2px (including negative space thickness)
+1. icons should have square rather than rounded corners where suitable
+
+### Technical:
+
+1. Icons must be SVG v1.1
+1. Icons should have a bounding box of 1024. This is because of a quirk with both versions of the Image Service, whereby a conversion from SVG to PNG will be very blurry if the _source_ SVG has a small viewBox.
```

Comment:
I don't think so, because it was a pre-existing bug in v1, so everything was already working round it. We should raise one though!